### PR TITLE
Updated supported options for docker create/run/start

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/276
Fixes https://github.com/vmware/vic-product/issues/273

### CODE REVIEW

**Comment by stuclem** [comment]
@anchal-agrawal when reviewing, please bear in mind that this update is for 1.1, not 1.2. Thanks!

File: docs/user_doc/vic_app_dev/container_operations.md [hunk 1]
```diff
@@ -55,8 +55,8 @@
...
+|`start`|[Start a container](https://docs.docker.com/engine/reference/commandline/start/)|Yes, since 1.0. Supports the `--attach`, `--cidfile`, `--cpuset-cpus`, `--entrypoint`, `--env`, `--env-file`, `--help`, `--interactive`, `--ip`, `--link`, `--memory`, `--name`, `--net`, `--net-alias`, `--publish`, `--rm`, `--stop-signal`, `--stop-timeout`, `--tty`, `--user`, `--volume`, and `--workdir` options.|
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_app_dev/container_operations.md]: Per the docker CLI v1.13 docs (https://docs.docker.com/v1.13/engine/reference/commandline/start/), all these options are not applicable to `start`. Out of the ones that are, we support `--attach` and `--interactive`.

**Comment by stuclem** [comment]
Thanks @anchal-agrawal. I removed everything from `docker start` apart from those two options.

**Comment by stuclem** [comment]
Thanks @anchal-agrawal !

**Pull request merged**